pkg/gofr/gofr_test.go

```diff
@@ -592,19 +582,51 @@ func TestStaticHandler(t *testing.T) {
 		}
 
 		if tc.expectedBodyLength != 0 {
-			contentLength, _ := strconv.Atoi(resp.Header.Get("Content-Length"))
-			assert.Equal(t, tc.expectedBodyLength, contentLength, "TEST [%d], Failed at Content-Length.\n %s", it, tc.desc)
+			contentLength := resp.Header.Get("Content-Length")
+			assert.Equal(t, strconv.Itoa(tc.expectedBodyLength), contentLength, "TEST [%d], Failed at Content-Length.\n %s", it, tc.desc)
 		}
 
 		if tc.expectedResponseHeaderType != "" {
 			assert.Equal(t,
 				tc.expectedResponseHeaderType,
 				resp.Header.Get("Content-Type"),
-				"TEST [%d], Failed at Expected Content-Type.\n%s",
-				it,
-				tc.desc)
+				"TEST [%d], Failed at Expected Content-Type.\n%s", it, tc.desc)
 		}
 
 		resp.Body.Close()
 	}
 }
+
+func createPublicDirectory(t *testing.T, htmlContent []byte) error {
```

Comment:
this is an helper, it should start with a `t.Helper()` call

https://pkg.go.dev/testing#T.Helper